streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -47,6 +47,7 @@
     private final ArrayDeque<ConsumerRecord<byte[], byte[]>> fifoQueue;
 
     private StampedRecord headRecord = null;
+    private long partitionTime = ConsumerRecord.NO_TIMESTAMP;
```

Comment:
Might be better to use `RecordQueue.UNKNOWN` instead of `NO_TIMESTAMP` (cf. `clear()` below)

Reply:
Sure, no problem with that.